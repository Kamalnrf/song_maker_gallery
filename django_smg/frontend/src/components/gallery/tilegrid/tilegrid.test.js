import React from "react";
import renderer from "react-test-renderer";
import TileGrid from "./tilegrid";

import { apiResponse } from "../../../test/__mocks__/api_response";

test("snapshot", () => {
  const tree = renderer
    .create(<TileGrid data={apiResponse["songData"][0]} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
