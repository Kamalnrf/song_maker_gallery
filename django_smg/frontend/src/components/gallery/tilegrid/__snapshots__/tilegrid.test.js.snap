// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  id="gallery"
>
  <div />
  <a
    href="#gallery_top"
  >
    <h2
      id={
        Array [
          "A Group of Marks",
        ]
      }
    >
      A Group of Marks
    </h2>
    <br />
    <span>
      â˜Ÿ
    </span>
  </a>
  <ul>
    <div>
      <div
        style={
          Object {
            "height": "103px",
            "width": "300px",
          }
        }
      >
        <a
          href="https://musiclab.chromeexperiments.com/Song-Maker/song/5676759593254912"
          rel="noopner noreferrer"
          target="_blank"
        >
          <svg
            viewBox="0 0 300 103"
            xmlns="https://www.w3.org/2000/svg"
          >
            <text
              dominantBaseline="middle"
              textAnchor="middle"
              x="50%"
              y="50%"
            >
              Mark J.
            </text>
          </svg>
        </a>
      </div>
    </div>
    <div>
      <div
        style={
          Object {
            "height": "103px",
            "width": "300px",
          }
        }
      >
        <a
          href="https://musiclab.chromeexperiments.com/Song-Maker/song/5676759593254912"
          rel="noopner noreferrer"
          target="_blank"
        >
          <svg
            viewBox="0 0 300 103"
            xmlns="https://www.w3.org/2000/svg"
          >
            <text
              dominantBaseline="middle"
              textAnchor="middle"
              x="50%"
              y="50%"
            >
              Mark J.
            </text>
          </svg>
        </a>
      </div>
    </div>
    <div>
      <div
        style={
          Object {
            "height": "103px",
            "width": "300px",
          }
        }
      >
        <a
          href="https://musiclab.chromeexperiments.com/Song-Maker/song/5676759593254912"
          rel="noopner noreferrer"
          target="_blank"
        >
          <svg
            viewBox="0 0 300 103"
            xmlns="https://www.w3.org/2000/svg"
          >
            <text
              dominantBaseline="middle"
              textAnchor="middle"
              x="50%"
              y="50%"
            >
              Mark J.
            </text>
          </svg>
        </a>
      </div>
    </div>
    <div>
      <div
        style={
          Object {
            "height": "103px",
            "width": "300px",
          }
        }
      >
        <a
          href="https://musiclab.chromeexperiments.com/Song-Maker/song/5676759593254912"
          rel="noopner noreferrer"
          target="_blank"
        >
          <svg
            viewBox="0 0 300 103"
            xmlns="https://www.w3.org/2000/svg"
          >
            <text
              dominantBaseline="middle"
              textAnchor="middle"
              x="50%"
              y="50%"
            >
              Mark J.
            </text>
          </svg>
        </a>
      </div>
    </div>
  </ul>
</div>
`;
